### PR TITLE
Create SECURITY.md

### PR DESCRIPTION
**Pull request by bryevdv**
cc @bokeh/dev 

Under GH new "Security Overview":

https://github.com/bokeh/bokeh/security

We are listed as needing to define a security policy. I've opened this PR to discuss the idea. 

Pros:

* Get the check mark for GH standards
* move tidelift info out of README
* tidelift will (supposedly) auto-detect https://forum.tidelift.com/t/coordinated-disclosure-plan-verification-automation/246/3

Cons/Questions

* how does this work with our branching? do we just keep the current branch up to date? Do we go back and update old branches?  Can we include a link to the version on HEAD and a note saying that is authoritative?
 
* What is our actual policy, re: security updates? current branch and latest one branch back?

As an aside, we have not been doing anything at all with `main`. Would it make sense to rebase and force-push main to the lastest release tag upon releases?

### CODE REVIEW

**Comment by mattpap** [comment]
> What is our actual policy, re: security updates? current branch and latest one branch back?

I suppose so. I doubt that we have resources to support much more than that.

**Comment by mattpap** [comment]
> As an aside, we have not been doing anything at all with main. Would it make sense to rebase and force-push main to the lastest release tag upon releases?

Do we actually need `main` branch? After cloning the active branch is the default branch of the repository, which is `branch-3.0` currently. I honestly don't see any use for it, though maybe I'm missing something.

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Security Policy
+
+## Supported Versions
+
+| Version | Supported          |
+| ------- | ------------------ |
+| 3.x     | :white_check_mark: |
```
Comment (mattpap) [inline SECURITY.md:7]: Is there semantic meaning associated with a *white* check mark, as opposed to e.g. a green one?

**Review by mattpap — APPROVED** [review]

File: SECURITY.md (hunk 1 above)
Comment (bryevdv) [reply SECURITY.md:7]: This is just what the GH default template creation process through the site UI used. I don't know if it is significant or not.

**Comment by bryevdv** [comment]
> I suppose so. I doubt that we have resources to support much more than that.

Yes just to be clear, I think that would be an *upper bound* on what we could support. 

> Do we actually need main branch? After cloning the active branch is the default branch of the repository, which is branch-3.0 currently. I honestly don't see any use for it, though maybe I'm missing something.

It's mostly just that it's a little unusual not to have one, and removing it may break some links somewhere. I think keeping the latest release on `main` is fairly common in situations like ours but I'm not necessarily opposed to just deleting it either.

File: SECURITY.md (hunk 1 above)
Comment (bryevdv) [reply SECURITY.md:7]: Screen Shot 2021-09-23 at 08 51 22

Best guess is that someone just thinks the solid background for the check is more accessible/easily decipherable

**Comment by bryevdv** [comment]
OK well if this seems fine as is I will go ahead an merge and make some discussions for the other questions that don't need to be resolved now.

**Pull request merged**